spec/models/block/layout_base_spec.rb

```diff
@@ -17,5 +17,9 @@
     it "builds blocks of the correct type" do
       expect(described_class.new(blocks_hash).blocks.first.type).to eq("big_number")
     end
+
+    it "sets blocks to full width by default" do
```

Comment:
The test description doesn't match the expectation.

Reply:
Yep, good catch - addressed in cf11d7fe099025e302c174fe06ab44485999d130